src/components/editor-page/app-bar/sync-scroll-buttons/buttonIcon.svg

```diff
@@ -1,20 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<svg
-        xmlns:dc="http://purl.org/dc/elements/1.1/"
-        xmlns:cc="http://creativecommons.org/ns#"
-        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-        xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-        xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-        xmlns="http://www.w3.org/2000/svg"
-        width="512"
-        height="512"
-        viewBox="0 0 135.46666 135.46666"
-        version="1.1"
-        id="svg8"
-        inkscape:version="1.0 (4035a4fb49, 2020-05-01)"
-        sodipodi:docname="buttonIcon.svg">
-    <defs
-            id="defs2"/>
+<!--
+SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+<svg xmlns:dc="http://purl.org/dc/elements/1.1/"
+     xmlns:cc="http://creativecommons.org/ns#"
```

Comment:
maybe this should be removed?

Reply:
Why

Reply:
Since we ~don't use creative commons~?  Nevermind. I didn't associate CC-0 with creative commons and if that were the case we probably don't want to reference two different lienceses.

Reply:
Maybe we want to add this then to the other svg files in the folder?

Reply:
I think this was in the file before. We could add the license to the other SVGs but that is out of scope here. Please create an issue.